packages/core/src/DreifussWysiwygEditor.tsx

```diff
@@ -132,6 +139,11 @@ export default function DreifussWysiwygEditor(props: EditorProps) {
           <ToolbarLink />
         </HeadingToolbar>
       )}
+      {props.showCharCount && (
+        <p style={{textAlign: 'right'}}>
+          {'Characters count:'} <CharCountToolbar />
```

Comment:
can you replace the text with an icon? Trying to avoid text because of translations.